README.md

```diff
@@ -1,5 +1,5 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
-# Fashion-Week-Panama
+# Fashion-Week
 This is a template for a Fashion and Clothing events page. The main objective is to showcase my skills in HTML/CSS, Bootstrap and JavaScript.
 ![screencapture-hunterx-7-github-io-Fashion-Week-2022-11-25-17_48_41](https://user-images.githubusercontent.com/90111138/204061316-5f7a9558-5934-42b1-a0d3-55537921c9db.png)
 
```

Comment:
- [ OPTIONAL ] In your next project, please make sure that you use the updated [README TEMPLATE](https://github.com/microverseinc/readme-template) :pray: